.github/workflows/ci_tests.yaml

```diff
@@ -31,14 +31,21 @@ jobs:
           python-version: 3.8
 
       - name: Install packages
-        run: pip install black blackdoc flake8 pylint
+        run: |
+          pip install black blackdoc flake8 pylint
+          sudo apt-get install dos2unix
 
       - name: Formatting check (black and flake8)
         run: make check
 
       - name: Linting (pylint)
         run: make lint
 
+      - name: Line breaks check (dos2unix)
+        run: |
+          find . -name "*.py" -exec dos2unix {} +
```

Comment:
Presumably we want to check all files, not just Python *.py files?

Reply:
Yes, but not sure what's the best/correct way to find all ASCII files.

Reply:
I just updated the command to:
```
find . -type f -not -path '*/\.git/*' -exec grep -Iq . {} \; -exec dos2unix {} \;
```
The command runs `dos2unix` on all ASCII files.